tests/lib/exporters/gce_disk_export.py

```diff
@@ -39,35 +39,42 @@ def testInitialization(self):
 
   # pylint: disable=line-too-long
   @mock.patch('libcloudforensics.providers.gcp.internal.compute.GoogleCloudCompute.GetDisk')
+  @mock.patch('dftimewolf.lib.collectors.gcloud.GoogleCloudCollector.SetUp')
```

Comment:
Why are we mocking GoogleCloudCollector if we're testing the exporter here?

Reply:
removed, was a residue from before, where I used a dummy gcloud collector to find instance disks, but now no need.